docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +255,216 @@ You should receive a response indicating there were no errors.
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "acknowledged": true,
+  "shards_acknowledged": true,
+  "index": "my-explicit-mappings-books"
+}
+----
+// TEST[skip:already created in setup]
+===============
+
+[discrete]
+[[getting-started-combined-mapping]]
+==== Combine dynamic and explicit mappings
 
-Indexed documents are available for search in near real-time.
+Explicit mappings are defined at index creation, and documents must conform to these mappings.
+When an index has the `dynamic` flag set to `true`, you can add new fields to documents without updating the mapping.
+This allows you to <<mapping-combine-explicit-dynamic,combine explicit and dynamic mappings>>.
 
 [discrete]
-[[search-all-documents]]
+[[getting-started-search-data]]
+=== Step 4: Search your index
+
+Indexed documents are available for search in near real-time, using the <<search-your-data,`_search` API>>.
+// TODO: You'll find more detailed quick start guides in TODO
+
+[discrete]
+[[getting-started-search-all-documents]]
 ==== Search all documents
 
 Run the following command to search the `books` index for all documents:
+
 [source,console]
 ----
 GET books/_search
 ----
 // TEST[continued]
 
-The `_source` of each hit contains the original
-JSON object submitted during indexing.
+.Expand to see (truncated) example response with annotations
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "took": 2, <1>
+  "timed_out": false, <2>
+  "_shards": { <3>
+    "total": 5,
+    "successful": 5,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": { <4>
+    "total": { <5>
+      "value": 7,
+      "relation": "eq"
+    },
+    "max_score": 1, <6>
+    "hits": [
+      {
+        "_index": "books", <7>
+        "_id": "CwICQpIBO6vvGGiC_3Ls", <8>
+        "_score": 1, <9>
+        "_source": { <10>
+          "name": "Brave New World",
+          "author": "Aldous Huxley",
+          "release_date": "1932-06-01",
+          "page_count": 268
+        }
+      },
+      ... (truncated)
+    ]
+  }
+}
+----
+// TEST[continued]
+<1> The `took` field indicates the time in milliseconds for {es} to execute the search
+<2> The `timed_out` field indicates whether the search timed out
+<3> The `_shards` field contains information about the number of <<glossary,shards>> that the search was executed on and the number that succeeded
+<4> The `hits` object contains the search results
+<5> The `total` object provides information about the total number of matching documents
+<6> The `max_score` field indicates the highest relevance score among all matching documents
+<7> The `_index` field indicates the index the document belongs to
+<8> The `_id` field is the document's unique identifier
+<9> The `_score` field indicates the relevance score of the document
+<10> The `_source` field contains the original JSON object submitted during indexing
+===============
 
 [discrete]
-[[qs-match-query]]
+[[getting-started-match-query]]
 ==== `match` query
```

Comment:
🤔 @georgewallace @shainaraskas @marciw  I'm wondering if we should even remove the match query here and make it the first query in the full-text search tutorial.

That way in this first tutorial we just present the `GET books/_search` call, as super basic "R" in "CRUD" and say "Learn more about search queries in foo tutorial". 🤷

Reply:
we'll revisit this line of thought once the next one is out the gate